web-ng/src/app/components/layer-list/layer-list.component.ts

```diff
@@ -47,16 +46,11 @@ export class LayerListComponent {
           List(project.layers.valueSeq().toArray()).sortBy(l => l.index)
         )
       );
-    this.initLayerListPermission();
   }
 
-  initLayerListPermission(): void {
-    const project = this.projectService.getCurrentProject();
-    if (!project) {
-      return;
-    }
-    const acl = this.projectService.getProjectAcl(project);
-    this.canAddLayer = this.authService.canManageProject(acl);
+  canShare(): boolean {
```

Comment:
Since is this is shared logic (no pun intended), we can also move this method into NavigationService (eg as `canShareProject()`). See previous comment.

Reply:
Unfortunately, this results in circular dependency: 
<img width="915" alt="Screen Shot 2021-02-02 at 10 42 53 PM" src="https://user-images.githubusercontent.com/27938664/106699181-fe942e80-65a7-11eb-8d15-0418900c8702.png">